src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj

```diff
@@ -6,6 +6,7 @@
     <RootNamespace>Microsoft.CodeAnalysis.Remote</RootNamespace>
     <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ApplyNgenOptimization Condition="'$(TargetFramework)' == 'net472'">partial</ApplyNgenOptimization>
```

Comment:
Question: we don't include the one targeting core in our package, is this condition required?

Reply:
I'd rather keep it even if it has no effect for `netcoreapp3.1` target - to be clear that we are not trying to embed IBC data on .NET Core (yet).

Reply:
OK, I will add this condition for other projects in my PR that include core bits in our vsix

Reply:
Wouldn't you want to actually _remove_ this condition in your PR, since we would want to use IBC data for .NET Core assemblies as well.

Reply:
if we can figure out how to do that, yes